wallet/connect/3rd-party-libraries/wagmi.md

```diff
@@ -15,61 +15,75 @@ the MetaMask browser extension and MetaMask Mobile.
 
 ## Prerequisites
 
-- Ensure you have a basic understanding of Ethereum smart contracts and React Hooks.
-- Set up a project with [Wagmi](https://wagmi.sh/react/getting-started).
-- Create an Infura API key and allowlist to [make read-only requests](../../how-to/make-read-only-requests.md).
+- [Node.js](https://nodejs.org/en/) version 20+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm/) version 9+
+- A text editor (for example, [VS Code](https://code.visualstudio.com/))
+- The [MetaMask extension](https://metamask.io/) or
+  [MetaMask Flask](/developer-tools/dashboard/get-started/create-api) installed
+- Basic knowledge of [React](https://react.dev/)
 
-## Configure the MetaMask connector
+## Steps
 
-In your Wagmi project, configure the MetaMask connector:
+### 1. Set up the project
 
-```javascript
+If you don't have an existing Wagmi project, set up a new project using Wagmi's `create wagmi` command
```

Comment:
I assume this must be done in your project folder? It's probably obvious to developers, so you can decide whether you want to clarify this or not.

EDIT: Reading below, this actually creates a project folder, so probably worth clarifying?

Reply:
It's pretty standard for a "create" CLI tool to create the project directory itself (e.g. Create React App), so I think most developers will already know. However, I tried clarifying the sentence a little.